rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -1020,15 +1019,9 @@ void GLGSRender::do_local_task(rsx::FIFO_state state)
 
 	rsx::thread::do_local_task(state);
 
-	if (state == rsx::FIFO_state::lock_wait)
```

Comment:
No, don't flip in lock wait. If you wish to flip create a new spin enum.
Flip needs to communicate externally which isn't particularly great for timing.